src/services/AuthService.js

```diff
@@ -2,31 +2,34 @@ import decode from 'jwt-decode'
 
 export default class AuthService {
 	constructor(domain) {
-		this.domain = 'http://localhost:3000'
+		this.domain = 'http://localhost:3001'
 	}
 
-	login = (credentials) => {
-		console.log(credentials)
+	login = (user) => {
+		console.log(user)
 		return this.authFetch(`${this.domain}/users/sign_in`, {
 			method: "POST",
-			body: JSON.stringify(credentials),
+			body: JSON.stringify(user),
 		})
 		.then(statusResponse => {
 			let token = statusResponse.headers.get('Authorization')
 			// set a JWT token in local storage, taken out of response from API
 			console.log(token);
 			this.setToken(token)
 			//return json from response
+			console.log(statusResponse);
 			return statusResponse.json()
 		})
 	}
 
 	register = (user) => {
+		console.log('HEREEEEEEEEEE - AuthService user', user)
```

Comment:
For WIP console.log is ok, but I'd be really careful.  Very easy to forget about these.